movie-browser/src/model/ApiReader.js

```diff
@@ -1,6 +1,16 @@
 const ApiReader = url => fetch(url)
-  .then(response => {
-    return response.json()
+  .then(async (response) => {
```

Comment:
async await <3

Reply:
Well, I was forced to during endless debugging.

Reply:
i love it